mkt/developers/views.py

```diff
@@ -373,12 +374,32 @@ def content_ratings_edit(request, addon_id, addon):
     }
     request.session.modified = True
 
-    return render(request, 'developers/apps/ratings/ratings_edit.html',
-                  {'addon': addon,
-                   'app_name': get_iarc_app_title(addon),
-                   'form': form,
-                   'company': addon.latest_version.developer_name,
-                   'now': datetime.now().strftime('%Y-%m-%d %H:%M:%S')})
+    ctx = {
+        'addon': addon,
+        'app_name': get_iarc_app_title(addon),
+        'form': form,
+        'company': addon.latest_version.developer_name,
+        'now': datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    }
+
+    if waffle.switch_is_active('iarc-upgrade'):
+        iarc_request_id = IARCRequestID.objects.filter(app_id=addon)
+        if iarc_request_id.exists():
```

Comment:
Why not use `get_or_create()` ? It tells you when it needed to create something so I think it'd be a nice replacement. Or at least a `.get()` in a `try`/`except` to avoid the rather ugly `iarc_request_id = iarc_request_id[0]` line below.